tools/upgrade.sh

```diff
@@ -69,6 +69,7 @@ then
   printf "${BLUE}${BOLD}%s ${UNDER}%s${RESET}\n" "Get your Oh My Zsh swag at:" "https://shop.planetargon.com/collections/oh-my-zsh"
 else
   printf "${RED}%s${RESET}\n" 'There was an error updating. Try again later?'
+  exit 1
 fi
 
 # Unset git-config values set just for the upgrade
```

Comment:
These lines below should still be called, so the exit condition would need to go after that.

Reply:
You’re right. I updated the PR so that we can still undo the temporary `autoStash` settings, but return the proper exit status:
https://github.com/ohmyzsh/ohmyzsh/blob/0a3bb71a36df6da563a4d1deaee437fa2a28e490/tools/upgrade.sh#L73-L84